go.mod

```diff
@@ -3,12 +3,11 @@ module github.com/getkin/kin-openapi
 go 1.16
 
 require (
-	github.com/go-openapi/jsonpointer v0.19.5
+	github.com/go-openapi/jsonpointer v0.19.6
 	github.com/gorilla/mux v1.8.0
-	github.com/invopop/yaml v0.1.0
+	github.com/invopop/yaml v0.2.0
 	github.com/mohae/deepcopy v0.0.0-20170929034955-c48cc78d4826
 	github.com/perimeterx/marshmallow v1.1.4
 	github.com/stretchr/testify v1.8.1
-	gopkg.in/yaml.v2 v2.4.0 // indirect
```

Comment:
What in this PR requires these upgrades?

If they are not needed by this PR I'd like these to be moved to another PR so we can move on them in parallel.

Reply:
removed if it is project way of handling decencies upgrade.